influxdb3_write/src/lib.rs

```diff
@@ -277,9 +285,43 @@ pub struct DatabaseTables {
     pub tables: hashbrown::HashMap<Arc<str>, Vec<ParquetFile>>,
 }
 
+/// The next file id to be used when persisting `ParquetFile`s
+pub static NEXT_FILE_ID: AtomicU64 = AtomicU64::new(0);
+
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq, Copy, Clone, PartialOrd, Ord)]
+/// A newtype wrapper for ids used with `ParquetFile`
+pub struct ParquetFileId(u64);
```

Comment:
This is a cool pattern, I wonder if we should adopt the atomic approach for our sequence IDs?